solidity/dashboard/src/components/Rewards.jsx

```diff
@@ -164,6 +167,10 @@ const RewardsComponent = ({ withdrawRewardAction }) => {
       : rewardsToReturn.slice(0, previewDataCount)
   }, [rewards, withdrawals, showAll, rewardFilter.status])
 
+  const amountTooltipText = (amount) => {
+    return `${displayEthAmount(amount, "Gwei", null)} Gwei`
```

Comment:
I think `gwei` all lowercase is “most correct”.

Reply:
Also, why pass `null` explicitly here?

Reply:
> Also, why pass null explicitly here?

Because in the `displayEthAmount` function the latest param(`decimalsPlaces`) has default value(`4`). Displaying the amount in `gwei`, we want to get the full amount without rounding to decimal places.

Reply:
> I think gwei all lowercase is “most correct”.

pls see 0bc3f45

Reply:
I'm going to ponder this some more over the weekend, but `null` feels like a problematic value to have to pass explicitly. It doesn't explain itself at all, which makes it harder to understand what exactly is happening. It feels like if `null`/`undefined` is a valid value, it should be the default, even if that means we're passing `4` in many other places.